Agenda/Implementation/Agenda/agenda.html

```diff
@@ -20,40 +20,40 @@
 </head>
 <body>
 
-  <!-- navbar start -->
+  <!-- The Entire Navbar Code -->
+
+  <!-- Primary Navbar (Black one) -->
   <nav class="navbar navbar-expand-lg navbar-dark">
-    <!-- When viewing on mobile, this will show up as the hamburger menu -->
-    <span class="navbar-brand">SCHEDULEARN</span>
+    
+    <!-- Navbar Menu Icon: Only visible in mobile version -->
     <button class="navbar-toggler" id="openNavigation" type="button" data-toggle="collapse" data-target="#navbarContent" aria-controls="navbarContent" aria-expanded="false" aria-label="Toggle navigation">
       <span class="navbar-toggler-icon"></span>
     </button>
 
+    <!-- Navbar Header -->
+    <span class="navbar-brand">SCHEDULEARN</span>
+
+    <!-- Navbar Menu Options: Only visible in desktop version -->
     <div class="collapse navbar-collapse">
       <ul class="navbar-nav mr-auto">
         <li class="nav-item active">
           <a class="nav-link" href="#">AGENDA<span class="sr-only">(current)</span></a>
         </li>
         <li class="nav-item">
-          <a class="nav-link" href="#">TUTURS</a>
+          <a class="nav-link" href="#">TUTORS</a>
         </li>
       </ul>
-      <div class="navbar-nav ml-auto">
-        <div id="profile-name" class="nav-item">
-          <a class="nav-link" href="{% url 'profile' %}">Ahmed Siddiqui</a>
-        </div>
-        <div id="pfp-container" class="nav-item dropdown">
-          <div id="profile-picture" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-          </div>
-          <div class="dropdown-menu dropdown-menu-right" aria-labelledby="profile-picture">
-            <button type="button" class="btn btn-danger">Sign Out</button>
-          </div>
-        </div>
-      </div>
     </div>
 
-  </nav>
+    <!-- Navbar Profile Icon: Visible in both desktop and mobile -->
+    <button class="bg-transparent border-0" id="openNavigation" type="button" data-toggle="collapse" data-target="#pfp-container" aria-controls="pfp-container" aria-expanded="false" aria-label="Toggle navigation">
+      <div id="profile-picture"></div>
+    </button>
+
+  </nav> <!-- PRIMARY NAVBAR CODE DOESN'T END HERE! -->
 
-  <div class="collapse navbar-collapse w-50 alignRight p-3 position-fixed rounded" id="navbarContent">
+  <!-- Navbar Menu: Collapsible menu only visible in mobile version -->
+  <div class="collapse navbar-collapse w-50 alignLeft p-3 position-fixed rounded" id="navbarContent">
```

Comment:
same thing - where is the use of id?

Reply:
This navbarContent is used by the button above for data toggle. The very first button in <nav>

Reply:
👍 sounds good